browser/app/js/browser/__tests__/actions.test.js

```diff
@@ -20,7 +20,7 @@ import * as actionsCommon from "../actions"
 
 jest.mock("../../web", () => ({
   StorageInfo: jest.fn(() => {
-    return Promise.resolve({ storageInfo: { Used: [60] } })
+    return Promise.resolve({ storageInfo: { Used: 60 } })
```

Comment:
@BigUstad In order to fix the testing failure, we would need to make the letter U in `User` lowercase here, as this is the value that's being mocked coming from the server.

```suggestion
    return Promise.resolve({ storageInfo: { used: 60 } })
```

We would additionally need to apply the following change, as the response object being returned from the `web.StorageInfo` info has the format `{storageInfo: {used: 60}}` instead of `{used: 60}`. If it is going to be a cleaner solution to return the info as the latter, a change in the backend might be applied instead:

```diff
diff --git a/browser/app/js/browser/actions.js b/browser/app/js/browser/actions.js
index f6850481..9621680a 100644
--- a/browser/app/js/browser/actions.js
+++ b/browser/app/js/browser/actions.js
@@ -32,9 +32,7 @@ export const closeSidebar = () => ({
 export const fetchStorageInfo = () => {
   return function(dispatch) {
     return web.StorageInfo().then(res => {
-      const storageInfo = {
-        used: res.used
-      }
+      const { storageInfo } = res
       dispatch(setStorageInfo(storageInfo))
     })
   }
```